addons/project/models/res_partner.py

```diff
@@ -80,3 +80,30 @@ def action_view_tasks(self):
         else:
             action['domain'] = search_domain
         return action
+
+    @api.model
+    def get_mention_suggestions_from_task(self, task_id, search, limit=8):
```

Comment:
Why not giving the `task_id` parameter to `self`? 🤔 🙂 

```suggestion
    def get_mention_suggestions_from_task(self, search, limit=8):
```

Reply:
ah right, my bad. 🙈

Reply:
ah right, my bad. 🙈